ikarus/solver/nonlinearsolver/newtonraphson.hh

```diff
@@ -115,21 +116,27 @@ public:
         dNorm       = norm(correction_);
         updateFunction_(x, correction_);
       }
-      this->notify(NonLinearSolverMessages::CORRECTIONNORM_UPDATED, static_cast<double>(dNorm));
-      this->notify(NonLinearSolverMessages::SOLUTION_CHANGED);
+      logInfo.correctionNorm = static_cast<double>(dNorm);
+      this->notify(NonLinearSolverMessages::CORRECTIONNORM_UPDATED, logInfo);
+      this->notify(NonLinearSolverMessages::SOLUTION_CHANGED, logInfo);
```

Comment:
I can already see the improvement. This is so much cleaner now!

Reply:
Thanks